charts/hivemq-platform/tests/hivemq_services_test.yaml

```diff
@@ -104,6 +104,58 @@ tests:
                   targetPort: mqtt-1884
                   port: 1884
 
+  - it: with additional labels
+    set:
+      services:
+        - type: mqtt
+          exposed: true
+          containerPort: 1884
+          labels:
+            test-label-key: test-label-value
+    asserts:
```

Comment:
should we assert it doesn't contain any annotation?

Reply:
That should already be tested using `content`. When I add an annotation to the setup, the assertion fails:
```
 FAIL  test HiveMQ Platform services    charts/hivemq-platform/tests/hivemq_services_test.yaml
        - with additional labels

                - asserts[1] `contains` fail
                        Template:       hivemq-platform/templates/hivemq-custom-resource.yml
                        DocumentIndex:  0
                        Path:   spec.services
                        Expected to contain:
                                - metadata:
                                    labels:
                                      test-label-key: test-label-value
                                    name: hivemq-test-hivemq-platform-mqtt-1884
                                  spec:
                                    ports:
                                      - name: mqtt-1884
                                        port: 1884
                                        targetPort: mqtt-1884
                        Actual:
                                - metadata:
                                    annotations:
                                      foo: bar
                                    labels:
                                      test-label-key: test-label-value
                                    name: hivemq-test-hivemq-platform-mqtt-1884
                                  spec:
                                    ports:
                                      - name: mqtt-1884
                                        port: 1884
                                        targetPort: mqtt-1884
                                - metadata:
                                    name: hivemq-test-hivemq-platform-metrics-9399
                                  spec:
                                    ports:
                                      - name: metrics-9399
                                        port: 9399
                                        targetPort: metrics-9399
```

Reply:
Sorry, I missed the use of `content`. My bad